.github/workflows/get_an_identity_data_sync_all.yml

```diff
@@ -0,0 +1,31 @@
+name: Resync User data for all GAI users
```

Comment:
How do we normally test this configuration? or other github actions? once on dev or something?

Reply:
Hm. They're tied tightly to the environment so we couldn't actually test them on a non-prod environment. This is a carbon copy of .github/workflows/update_all_schools.yml with the rake task swapped out which runs daily, so I would be confident this structure works.

I'm going to manually run the rake task itself in dev to test that there though.

Reply:
Yeah sounds good, more an observation i guess